src/p5xr/core/p5xrviewer.js

```diff
@@ -48,3 +54,19 @@ p5.prototype.setViewerPosition = function(x, y, z) {
   let viewer = p5xr.instance.viewer;
   viewer.setPosition(x, y, z);
 };
+
+p5.prototype.sticky = function(drawOnTop = false) {
+  push();
+  p5xr.instance.viewer.drawOnTop = drawOnTop;
+  if(drawOnTop)
+    p5.instance._renderer.GL.disable(p5.instance._renderer.GL.DEPTH_TEST);
+  p5.instance._renderer.uMVMatrix.set(p5.Matrix.identity());
+  let viewerPosition = p5xr.instance.viewer.position;
+  setViewerPosition(viewerPosition.x, viewerPosition.y, viewerPosition.z);
+};
+
+p5.prototype.noSticky = function() {
```

Comment:
The use of `push` and `pop` here might warrant some safe guards. What happens if a user calls `sticky` but not `noSticky` in the draw loop? The model transforms will reset at the start of the loop but the state stack will not. Perhaps there needs to be some property on p5xr that knows whether the mode is `sticky` and `pop` that state at the end of the draw loop?

Reply:
The model transforms will be reset anyways in the next draw loop. But there might be other properties which will not be restored. How can we detect end of draw loop? Perhaps users should make sure they call `noSticky()` appropriately?

Reply:
I think that is okay for the time being.